motr/client.h

```diff
@@ -917,6 +917,8 @@ enum
 
 enum m0_pi_calc_flag {
```

Comment:
consider a case where in user passes flag by ORing
ie flag = M0_PI_CALC_UNIT_ZERO | M0_PI_SKIP_CALC_FINAL
here the value of flag is 3 (enum doesn't have value 3)

Reply:
flag = M0_PI_CALC_UNIT_ZERO | M0_PI_SKIP_CALC_FINAL
flag value = 3 or 11 in binary

M0_PI_CALC_UNIT_ZERO - 01
M0_PI_SKIP_CALC_FINAL - 10

11 & 01 - true
11 & 10 - true

We do not need to add another flag for 3.